Platform/MeteorlakeBoardPkg/Script/StitchIfwiConfig_mtlp.py

```diff
@@ -135,12 +135,12 @@ def get_xml_change_list (platform, plt_params_list):
       #Region Order
         ('./FlashLayout/BiosRegion/InputFile',                                      '$SourceDir\BiosRegion.bin'),
         ('./FlashLayout/DescriptorRegion/HarnessGlobalData/SelectedRvp',            'MTL-P DDR5 (MTL-P + MTP-P)'),
-        ('./FlashLayout/Ifwi_IntelMePmcRegion/MeRegionFile',                        '$SourceDir\MeRegionFile.bin'),
-        ('./FlashLayout/Ifwi_IntelMePmcRegion/PmcBinary',                           '$SourceDir\PmcBinary.bin'),
-        ('./FlashLayout/EcRegion/InputFile',                                        '$SourceDir\EcRegion.bin'),
+        ('./FlashLayout/Ifwi_IntelMePmcRegion/MeRegionFile',                        '$SourceDir\ME Sub Partition.bin'),
+        ('./FlashLayout/Ifwi_IntelMePmcRegion/PmcBinary',                           '$SourceDir\CsePlugin#PMC.bin'),
+        ('./FlashLayout/EcRegion/InputFile',                                        '$SourceDir\SystemPlugin#EcRegion.bin'),
```

Comment:
The original name is same with the path node name and it could be handled by tool easily. User could get to know the file details from help info in FIT tool.
It looks the new name is from the decomposed IFWI image, but we expect user to get these ingredients from individual release package. The new name might mislead the user.

Reply:
as long as user follow the release note then it should be fine for them right

Reply:
the intention of renaming this is to increase the efficiency for us during SBL IFWI integration

Reply:
AFAIK, in SBL release note we used FIT node name in all the projects. If you get these ingredients from individual package, I didn't see any efficiency difference. For internal test purpose to get the ingredients from BIOS IFWI, we have an internal tool (GenXmlDiff.py) to help decompose it and generate these binary files whose name follows FIT path node (so it is more efficiency).

Reply:
will proceed to close this PR